animalai/play.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+from pathlib import Path
+
+from animalai import AnimalAIEnvironment, arenas
+from animalai.executable import find_executable
```

Comment:
Grouping the modules will reduce lines of code (LoC), a standard in industry to aim for.

```suggestion
import random
from pathlib import Path
from animalai import AnimalAIEnvironment, arenas
from animalai.executable import find_executable
```